.vitepress/theme/components/VueJobs.vue

```diff
@@ -1,14 +1,29 @@
 <script lang="ts">
 // shared data across instances so we load only once
 const base = `https://app.vuejobs.com/feed/vuejs/docs?format=json`
-let items = $ref([])
+
 </script>
 
 <script setup lang="ts">
 import { onMounted, computed } from 'vue'
 
 let vuejobs = $ref<HTMLElement>()
 
+let items = $ref<Jobs[]>([])
+
+type Jobs = {
+  organization: Organization;
+  title: string;
+  link: string;
+  locations: Array<string>;
+  remote: false | "ALLOWED" | "ONLY";
+};
+
+type Organization = {
+  name: string;
+  avatar: string; 
```

Comment:
it can be null so we should choose an avatar to show whenever user didn't upload any image.